apps/qubit/modules/physicalobject/config/security.yml

```diff
@@ -1,12 +1,17 @@
+index:
+  credentials: [[ editor, administrator, translator ]]
+  is_secure: true
```

Comment:
It's not necessary to specify `is_secure: true` for each action, it's sufficient that it's specified for the "all:" permissions block.  I would also move the "all:" block to the top of the security list, just for legibility but it's not necessary.

Reply:
Ref: https://symfony.com/legacy/doc/reference/1_4/en/08-Security